### PR TITLE
Add possibility to retry a pbl payment

### LINKED ISSUE #71: Support retrying a `Pay by link` payment

### PR DESCRIPTION
**Pull request by arti0090**
Closes #71

Adds coverage for #96 🚀

### CODE REVIEW

File: tests/E2E/Order/TpayRetryOrChangePaymentOrderTest.php [hunk 1]
```diff
@@ -5,6 +5,7 @@
 namespace Tests\CommerceWeavers\SyliusTpayPlugin\E2E\Order;
 
 use Facebook\WebDriver\WebDriverBy;
+use Facebook\WebDriver\WebDriverDimension;
```
Comment (lchrusciel) [inline tests/E2E/Order/TpayRetryOrChangePaymentOrderTest.php:8]: not needed

**Review by lchrusciel — APPROVED** [review]
code wise 👍

File: config/config/sylius_template_events.php [hunk 2]
```diff
@@ -17,6 +17,10 @@
                         'template' => '@CommerceWeaversSyliusTpayPlugin/shop/order/pay/_card.html.twig',
                         'priority' => 10,
                     ],
+                    'pay-by-link' => [
```
Comment (jakubtobiasz) [inline config/config/sylius_template_events.php:20]:
```suggestion
                    'pay_by_link' => [
```

to match the line `36`

**Review by jakubtobiasz — APPROVED** [review]

**Comment by lchrusciel** [comment]
I can confirm, it does not work as expected

**Comment by jakubtobiasz** [comment]
I can confirm it works after the latest fix.

**Review by jakubtobiasz — APPROVED** [review]

**Pull request merged**